docs/configuration.md

```diff
@@ -75,6 +75,7 @@ Unless you need to set a non-default value, it is recommended to only populate o
 | `HBAR_RATE_LIMIT_BASIC`                     | "1120000000"               | Individual limit (in tinybars) for spending plans with a BASIC tier. Defaults to 11.2 HBARs.                                                                                                                                                                                                           |
 | `HBAR_RATE_LIMIT_EXTENDED`                  | "3200000000"               | Individual limit (in tinybars) for spending plans with a EXTENDED tier. Defaults to 32 HBARs.                                                                                                                                                                                                          |
 | `HBAR_RATE_LIMIT_PRIVILEGED`                | "8000000000"               | Individual limit (in tinybars) for spending plans with a PRIVILEGED tier. Defaults to 80 HBARs.                                                                                                                                                                                                        |
+| `HBAR_SPENDING_PLANS_CONFIG_JSON`           | "" | Spending plans can be loaded from environment varaibles in JSON format, based on the spendingPlansConfig.example.json file.                                    |
```

Comment:
Is it possible to reuse `SPENDING_PLANS_CONFIG_FILE` and rename it to something more general?
In that way if the logic realizes it's a json file it'll read it, if not it'll parse JSON text.
Could this work? Are there tradeoffs?
I'm mindful of the number of configs and the result of both cases is a valid spending plan

Reply:
It's renamed to simply `SPENDING_PLANS_CONFIG`.  Yes it can work, and yes there are trade offs.

**Pros**
- simplicity
- flexibility
- maintainability

**Cons**
- ambiguity:  An input string could validly be both a file path and a JSON string.  Tests cover both scenarios and JSON content is validated when it's parsed.
- Performance Overhead: Checking if a file exists (fs.existsSync) introduces additional I/O operations, however this happens only once at start up.